linux/sound.cpp

```diff
@@ -33,14 +33,18 @@ void CSound::OpenJack ( const bool  bNoAutoJackConnect,
     jack_status_t JackStatus;
 
     // try to become a client of the JACK server
-    pJackClient = jack_client_open ( jackClientName, JackNullOption, &JackStatus );
+    pJackClient =
+        jack_client_open ( jackClientName, JackNullOption, &JackStatus );
 
     if ( pJackClient == nullptr )
     {
-        throw CGenErr ( tr ( "The Jack server is not running. This software "
+        throw CGenErr ( tr (
```

Comment:
It's been a bit erratic in its long string reflows.  Splitting like it has but then not joining isn't nice.

Reply:
doesn't split with `ColumnLimit: 120`